packages/sidenav/src/sidenav.ts

```diff
@@ -99,7 +99,8 @@ export class SideNav extends Focusable {
 
     private focusItemByOffset(direction: number): void {
         const items = [...this.querySelectorAll('sp-sidenav-item')];
-        const focused = items.indexOf(document.activeElement as SideNavItem);
+        const focused = items.indexOf((this.getRootNode() as Document)
```

Comment:
Would it be possible to have `(this.getRootNode() as Document).activeElement` as a helper? I could see this as a reusable pattern

Reply:
What would you see a helper here looking like? Like a `getActiveElement(this)` sort of thing? 

I could also see it being used more regularly, but does that make the results seems more "magic" than the reality of the DOM APIs that they are based on?

Reply:
yeah, thats sort of what I had in mind. I was thinking it would be beneficial specifically for the shorthand convenience and improving the readability/prettiness of the code (less typecasting on the same line)